apps/admin-portal/src/components/identity-providers/wizard/steps/general-settings.tsx

```diff
@@ -16,7 +16,7 @@
  * under the License.
  */
 
-import React, { FunctionComponent } from "react";
+import React, {FunctionComponent, useEffect} from "react";
```

Comment:
```suggestion
import React, { FunctionComponent, useEffect } from "react";
```

Reply:
Fixed in 6311ed7b87d8256e38389b5145f845d46e4c8564.